### PR TITLE
fix for bubble merge and split repeat

### PR DESCRIPTION
**Pull request by jakebiesinger**
@anbangx @Nan-Zhang @JavierJia @Elmira88 
I found and squashed a bug in bubble merge and two in split repeat.  Those patterns seem to be working now.  I'll generate some metrics to be sure, but these should be good to merge in now!

### CODE REVIEW

File: genomix/genomix-pregelix/src/main/java/edu/uci/ics/genomix/pregelix/operator/splitrepeat/SplitRepeatVertex.java [hunk 1]
```diff
@@ -75,6 +75,7 @@ public String generaterRandomDNAString(int n, String vertexId) {
         StringBuilder sb = new StringBuilder();
         // The maximum edge number of one vertex is 8, so 4**num_letters is always bigger than len(existing)
         while (true) { // impossible infinite loop
+            sb = new StringBuilder();
```
Comment (anbangx) [inline genomix/genomix-pregelix/src/main/java/edu/uci/ics/genomix/pregelix/operator/splitrepeat/SplitRepeatVertex.java:78]: The initiation outside the loop is unnecessary?

**Comment by anbangx** [comment]
LGTM!

File: genomix/genomix-pregelix/src/main/java/edu/uci/ics/genomix/pregelix/operator/splitrepeat/SplitRepeatVertex.java (hunk 1 above)
Comment (jakebiesinger) [inline genomix/genomix-pregelix/src/main/java/edu/uci/ics/genomix/pregelix/operator/splitrepeat/SplitRepeatVertex.java:78]: Yeah that's true I guess.

**Comment by Nan-Zhang** [comment]
Good work! jake, LGTM~

File: genomix/genomix-pregelix/src/main/java/edu/uci/ics/genomix/pregelix/operator/splitrepeat/SplitRepeatVertex.java (hunk 1 above)
Comment (jakebiesinger) [inline genomix/genomix-pregelix/src/main/java/edu/uci/ics/genomix/pregelix/operator/splitrepeat/SplitRepeatVertex.java:78]: Done.

**Pull request merged**